VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -486,4 +486,14 @@
   <data name="FlagsEnumValuesAreNotPowersOfTwoValuesDontFitAnalyzerTitle" xml:space="preserve">
     <value>[Flags] enum its values are not explicit powers of 2 and its values dont fit in the specified enum type.</value>
   </data>
+  <data name="SyncMethodWithSyncSuffixAnalyzerMessage" xml:space="preserve">
+    <value>Method "{0}" should not end with 'Async'.</value>
```

Comment:
I would specify this as 'Synchronous method (..)' to make it clear why it should be done. You could also update the other analyzer's message to specify the opposite: 'Asynchronous method (..)'